# Dual Servo Control Message
# Controls 2 servos independently using right analog stick

int32 servo1_speed  # Servo 1 speed in microseconds (1000-2000, 1500=stop)
int32 servo2_speed  # Servo 2 speed in microseconds (1000-2000, 1500=stop)
